string robot_name
float32 velocity
float32 distance